guides/source/debugging_rails_applications.md

```diff
@@ -143,7 +143,7 @@ Rails.logger.level = 0 # at any time
 
 This is useful when you want to log under development or staging without flooding your production log with unnecessary information.
 
-TIP: The default Rails log level is `debug` in all environments.
+TIP: The default Rails log level is `debug` for `development` and `test` environments. And `info` for the `production` environment.
```

Comment:
To be more precise, the default is always `:debug`, but it is overridden [in `config/environments/production.rb`](https://github.com/rails/rails/blob/663e8e1ccd049dc005924f7033ae407d2d25ba8a/railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt#L61).  So what about something like:

```suggestion
TIP: The default Rails log level is `:debug`. However, it is set to `:info` for the `production` environment in the default generated `config/environments/production.rb`.
```